hack/build.sh.d/kn-plugin-workflow.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Env variables can be injected to override values during build phase
+readonly quarkus_version="${LOGIC_QUARKUS_VERSION:-2.13.8.SP2-redhat-00001}"
+readonly quarkus_platform_group_id="${LOGIC_QUARKUS_GROUP:-com.redhat.quarkus.platform}"
+readonly dev_mode_image="${LOGIC_DEV_MODE_IMAGE:-registry.redhat.io/openshift-serverless-1-tech-preview/logic-swf-devmode-rhel8:1.31}"
```

Comment:
@Kaustubh-pande FYI, if we need to inject different values during build phase.

Reply:
Does we need to inject dev_mode_image build ref? or its just for the future.

Reply:
So far it was fine with floating tag, for the next release `:1.31. Depends on Logic team if they would like to use digest or not. For the future and our use case,it's also handy, e.g. for patches unrelated to our code like Quarkus version.